adding-markdown-support.rst

```diff
@@ -0,0 +1,74 @@
+.. post:: Oct 16, 2015
+   :tags: commonmark, markdown, feature
+
+Read the Docs & Sphinx now support Commonmark
+=============================================
+
+Read the Docs has long been built on top of Sphinx as it's primary technology.
+Sphinx for a long time has been built on top of reStructuredText as it's primary input mechanism.
```

Comment:
These two senetences are fragmenty on their own, consider joining them. Also, "long been/time" feels unsure. "Read the Docs was built on top of Sphinx, which has always relied on rST as an input mechanism." Though maybe that's not all true